.travis.yml

```diff
@@ -1,11 +1,11 @@
-language: generic
-cache: bundler
+language: ruby
```

Comment:
@nnat425 do you remember why we used "generic" here before?  I thought it was essential for Travis to work, but I don't remember the specifics.

Reply:
@LucasCioffi, `language: generic` was initially set up by @kfriedman. Originally, travis returns failures when `language: ruby` was set.

Reply:
Just so this is noted: In order for the caching to work, I learned that the language needed to be switched to ruby. Otherwise, it would throw me errors on generic.